packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/__tests__/LivenessCheck.test.tsx

```diff
@@ -144,4 +168,19 @@ describe('LivenessCheck', () => {
     expect(screen.getByText(landscapeMessageText)).toBeInTheDocument();
     expect(screen.queryByText('LivenessCameraModule')).not.toBeInTheDocument();
   });
+
+  it('should render the component on mobile', () => {
+    mockActorState.matches.mockReturnValue(true);
+    (global.navigator as any).userAgent =
```

Comment:
What does this do?

Reply:
whoops, combined this with the other unit test but this should test that on mobile we can detect landscape mode correctly and show an error